doc/api/http.md

```diff
@@ -744,15 +744,12 @@ The `encoding` argument is optional and only applies when `chunk` is a string.
 Defaults to `'utf8'`.
 
 The `callback` argument is optional and will be called when this chunk of data
-is flushed.
+is flushed, but only if the chunk is non-empty.
 
 Returns `true` if the entire data was flushed successfully to the kernel
 buffer. Returns `false` if all or part of the data was queued in user memory.
 `'drain'` will be emitted when the buffer is free again.
 
-When `write` function is called with empty string or buffer, it does
-nothing and waits for more input.
```

Comment:
I would keep this and the empty line afterwards.

Reply:
Added back.